gcp/modules/gcp-stackdriver-export/gpii_key.tf

```diff
@@ -0,0 +1,22 @@
+provider "google" {
+  project     = "${var.project_id}"
+  credentials = "${var.serviceaccount_key}"
+}
+
+resource "google_bigquery_dataset" "this" {
```

Comment:
perhaps "this" is not a good name for a resource.

Reply:
I don't see why, but ok, lets change it to something more meaningful.